crud/get.lua

```diff
@@ -72,23 +56,59 @@ function get.call(space_name, key, opts)
     -- async, there could be a lag between master and replica, so a
     -- connector which sequentially calls put() and then get() may get
     -- a stale result.
-    local result, err = call.rw_single(
+    local storage_result, err = call.rw_single(
         bucket_id, GET_FUNC_NAME,
         {space_name, key}, {timeout=opts.timeout})
 
     if err ~= nil then
-        return nil, GetError:new("Failed to get: %s", err)
+        return nil, GetError:new("Failed to call get on storage-side: %s", err)
     end
 
+    if storage_result.err ~= nil then
+        return nil, GetError:new("Failed to get: %s", storage_result.err)
+    end
+
+    local tuple = storage_result.res
+
     -- protect against box.NULL
-    if result == nil then
-       result = nil
+    if tuple == nil then
+        tuple = nil
```

Comment:
`_tuple` -> `tuple`

Reply:
No, indentation was wrong, but now it's OK.